### ISSUE TITLE
The Editor shows TypeError: Cannot read properties of null (reading 'toString') when Gutenberg is active

### ISSUE THREAD

**Issue by RafaelFunchal** [Automattic/wp-calypso#97528]
### Quick summary

Slack ping: p1734361511221669-slack-C03TY6J1A

When Gutenberg is active on the site, it is not possible to edit and create posts and pages.

The Editor shows the modal:

> The editor has encountered an unexpected error.

Copying the error reveals these lines:

```
TypeError: Cannot read properties of null (reading 'toString')
    at https://DOMAIN/wp-content/plugins/gutenberg/build/components/index.min.js?ver=1d11f77954d9898b0db4:94:6274
    at t (https://DOMAIN/wp-content/plugins/gutenberg/build/components/index.min.js?ver=1d11f77954d9898b0db4:1:2840)
    at n (https://DOMAIN/wp-content/plugins/gutenberg/build/components/index.min.js?ver=1d11f77954d9898b0db4:94:9945)
    at t.get (https://DOMAIN/wp-content/plugins/gutenberg/build/components/index.min.js?ver=1d11f77954d9898b0db4:94:10106)
    at t.render (https://DOMAIN/wp-content/plugins/gutenberg/build/components/index.min.js?ver=1d11f77954d9898b0db4:94:18759)
    at kr (https://DOMAIN/wp-content/plugins/gutenberg/build/vendors/react-dom.min.js?ver=18:1:59358)
    at br (https://DOMAIN/wp-content/plugins/gutenberg/build/vendors/react-dom.min.js?ver=18:1:59157)
    at js (https://DOMAIN/wp-content/plugins/gutenberg/build/vendors/react-dom.min.js?ver=18:1:121074)
    at Sl (https://DOMAIN/wp-content/plugins/gutenberg/build/vendors/react-dom.min.js?ver=18:1:88331)
    at kl (https://DOMAIN/wp-content/plugins/gutenberg/build/vendors/react-dom.min.js?ver=18:1:88259)
```

The domain has been removed for privacy concerns.

We already tried deactivating all plugins and switching to a Core theme.

### Steps to reproduce

1. Go to Posts
2. Click on Add New Post or on any post to edit.
3. The error is displayed.

### What you expected to happen

To load the editor to create and edit posts.

### What actually happened

The editor shows an error:

> The editor has encountered an unexpected error.

### Impact

One

### Available workarounds?

Yes, easy to implement

### If the above answer is "Yes...", outline the workaround.

For now, the workaround is to deactivate the Gutenberg plugin.

**Comment by RafaelFunchal** [comment]
@Robertght , unfortunately, the issue still happens with Gutenberg:19.9.0.

I created a post on the Happiness Feedback P2 as recommended: p2EDhh-2on-p2

**Comment by ramonjd** [comment]
I also can't reproduce. Does it occur with any theme, and with all plugins (except Gutenberg) disabled?

**Comment by tyxla** [comment]
Can't reproduce either.

> We may also need to create a replica of their website, so setting up a test site for our Gutenberg team to test could be good.

This could be very helpful, thanks 👍

**Comment by RafaelFunchal** [comment]
>  Does it occur with any theme, and with all plugins (except Gutenberg) disabled?

Yup! With only Jetpack (it can't be disabled on WP.com) and Twenty Twenty-Five, the editor breaks after reactivating Gutenberg.

> We may also need to create a replica of their website, so setting up a test site for our Gutenberg team to test could be good.

I'm trying to create a site copy on my account, but uploading the 6.5GB backup via SFTP takes hours. I will share the site link tomorrow (I'm EOD).

I shared the link to download a VaultPress backup in this comment p2EDhh-2on-p2#comment-9275 if you want to download it and test the site on different environments. I couldn't replicate after importing the backup to the Studio app, though.

**Comment by RafaelFunchal** [comment]
I shared the site link and username to use on this commend: p2EDhh-2on-p2#comment-9276

**Comment by tyxla** [comment]
For some reason, I'm still unable to reproduce it, see my comment here: p2EDhh-2on-p2#comment-9277

**Comment by mgozdis** [comment]
Additional user report here: 9419149-zd-a8c

We aren't sure how to reproduce and the same issue exists on their staging site we were testing on. The editor only loads if we either deactivate all plugins or deactivate Gutenberg. Switching to an unmanaged version of Gutenberg to update to the latest doesn't help. 

We can't determine a specific plugin conflict as sometimes the editor also works fine when reactivating some of the plugins. It breaks sporadically again later and remains broken until deactivating all user-installed plugins again.

**Comment by janmckell** [comment]
@draganescu Hi there.. Seeking your advice on whether to move this issue to the Gutenberg repo.
[A similar issue](https://github.com/WordPress/gutenberg/issues/53863) seem to have been fixed at version 16.5, but if it is the same issue and still occuring at version 20 (at minimum), would you advise that a new report be filed in the Gutenberg repo ? The issue seems difficult to consistently reproduce as well. Do you have any guidance on steps for testing that can be used to file a Gutenberg report, if needed ?

**Comment by Robertght** [comment]
I imported the site into Studio and wasn't able to replicate it here either.

I think we spent a lot of time on this and since we can't replicate it, I'm going to close it, but please reopen if other cases are ocurring.

**Comment by happykeyflow** [comment]
Another user report here: 9619737-zd
